pwnlib/tubes/sock.py

```diff
@@ -112,6 +112,10 @@ def can_recv_raw(self, timeout):
             self.recv_raw(1, socket.MSG_PEEK)
         except EOFError:
             return False
+        # SSLSocket and Paramiko recv() do not support any flags
+        except (ValueError, TypeError):
+            # select() said there is data, so pass
+            pass
```

Comment:
This should be in a different pull request

Reply:
Ok, sure, please tell me how you want me to proceed so I don't create greater mess.

Reply:
Can you add a doctest for this case>